src/components/ResourcesCard.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import InformationCards from "./cards/InformationCards";
+import HeaderCArd from "./cards/HeaderCArd";
+
+interface Props {
+  Entries: any;
+}
+
+const ResourcesCard = ({ Entries }: Props) => {
+  const { title, description, entries, colSpan, rowSpan, heightMultiplier } =
+    Entries;
+  return (
+    <div
+      className={`col-span-${colSpan} row-span-${rowSpan} p-8 bg-gray-100 border-2 border-dashed border-gray-400`}
+    >
+      <HeaderCArd title={title} description={description} />
+      <div className="m-auto">
+        <InformationCards Data={entries} heightMultiplier={heightMultiplier} />
```

Comment:
props should always be written in `camelCase`

Reply:
`heightMultiplier` is not a descriptive prop name. I would name this prop using a different name which would be more descriptive

Reply:
fixed